core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -167,8 +171,9 @@ trait MethodConfigurationComponent {
         }
     }
 
-    def hideMethodConfigurationAction(id: Long, name: String): ReadWriteAction[Int] = {
-      findById(id).map(rec => (rec.deleted, rec.name)).update(true, name + "_" + DateTime.now().toString("yyyy-MM-dd_HH:mm:ss"))
+    def hideMethodConfigurationAction(id: Long, methodConfigName: String): ReadWriteAction[Int] = {
+      findById(id).map(rec => (rec.deleted, rec.name))
+        .update(true, methodConfigName + "_" + DateTime.now().toString("yyyy-MM-dd_HH:mm:ss"))
```

Comment:
And does trashing the timestamp mean having to migrate existing data? I don't think so...